src/locales/en/translation.json

```diff
@@ -4,6 +4,9 @@
     "guild": "Guild",
     "guilds": "Guilds"

```

Comment:
You can remove this key

Reply:
I think we can keep these here for now since we may still use elements in future using the language of guilds